packages/block-library/src/paragraph/edit.native.js

```diff
@@ -99,8 +94,6 @@ class ParagraphEdit extends Component {
 			content,
 		} = attributes;
 
-		const minHeight = styles.blockText.minHeight;
```

Comment:
It's not clear to me why `minHeight` is not used in here in `Para` block, instead it's still used in `Heading` and `Title`.

Reply:
I was testing if we actually needed this, and find out that we don't... we don't need to force a minHeight the component will size itself to whatever text size.

Reply:
I removed the minHeight from all blocks that were using it. So that way the block code will be less aware of the platform.